### PR TITLE
feat: accept dispute

### PR DESCRIPTION
**Pull request by Riddhiagrawal001**
## Type of Change

- [ ] Bugfix
- [X] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description
image

## Checklist

- [x] I ran `npm run re:build`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/screens/HyperSwitch/Disputes/ShowDisputes.res [hunk 1]
```diff
@@ -47,21 +107,78 @@ module DisputesInfo = {
...
+
+      let handleAcceptDispute = async () => {
+        try {
+          let url = getURL(
+            ~entityName=ACCEPT_DISPUTE,
+            ~methodType=Post,
+            ~id=Some(data.dispute_id),
+            (),
+          )
+          let _response = await updateDetails(url, Dict.make()->Js.Json.object_, Post)
```
Comment (PritishBudhiraja) [inline src/screens/HyperSwitch/Disputes/ShowDisputes.res:123]: Put _

Comment (Riddhiagrawal001) [reply src/screens/HyperSwitch/Disputes/ShowDisputes.res]: Resolved

**Review by PritishBudhiraja — APPROVED** [review]

File: src/screens/HyperSwitch/Disputes/ShowDisputes.res [hunk 2]
```diff
@@ -33,63 +33,184 @@ module DisputesNoteComponent = {
...
-        </FormRenderer.DesktopRow>
-        <UIUtils.RenderIf condition={children->Option.isSome}>
-          {children->Option.getWithDefault(React.null)}
         </UIUtils.RenderIf>
-      </OrderUtils.Section>
-    }
+      </div>
+      <div className="h-px w-full bg-grey-200 opacity-30" />
+      <UIUtils.RenderIf
+        condition={disputeEvidenceUpload && data.dispute_status === "dispute_opened"}>
```
Comment (PritishBudhiraja) [inline src/screens/HyperSwitch/Disputes/ShowDisputes.res:174]: Ask backend for types so we can create a mapper

**Review by PritishBudhiraja — DISMISSED** [review]

File: src/screens/HyperSwitch/Disputes/ShowDisputes.res (hunk 2 above)
Comment (Riddhiagrawal001) [reply src/screens/HyperSwitch/Disputes/ShowDisputes.res]: Addressed

**Review by JeevaRamu0104 — DISMISSED** [review]

File: src/screens/HyperSwitch/Disputes/ShowDisputes.res [hunk 3]
```diff
@@ -33,63 +33,194 @@ module DisputesNoteComponent = {
...
-        <UIUtils.RenderIf condition={children->Option.isSome}>
-          {children->Option.getWithDefault(React.null)}
         </UIUtils.RenderIf>
-      </OrderUtils.Section>
-    }
+      </div>
+      <div className="h-px w-full bg-grey-200 opacity-30" />
+      <UIUtils.RenderIf
+        condition={disputeEvidenceUpload &&
+        showDisputeInfoStatus->Js.Array2.includes(
```
Comment (PritishBudhiraja) [inline src/screens/HyperSwitch/Disputes/ShowDisputes.res:182]: Array.includes

File: src/screens/HyperSwitch/Disputes/ShowDisputes.res [hunk 4]
```diff
@@ -33,63 +33,194 @@ module DisputesNoteComponent = {
...
-                    overiddingHeadingStyles="text-black text-sm font-medium"
-                    textColor="!font-normal !text-jp-gray-700"
-                  />
-                </div>
-              </UIUtils.RenderIf>
-            })
-            ->React.array}
+        <UIUtils.RenderIf
+          condition={disputeEvidenceUpload &&
+          showDisputeInfoStatus->Js.Array2.includes(
```
Comment (PritishBudhiraja) [inline src/screens/HyperSwitch/Disputes/ShowDisputes.res:158]: Array.includes

**Review by PritishBudhiraja — APPROVED** [review]

**Pull request merged**